geckodriver.log

```diff
@@ -932,3 +932,69 @@ console.warn: SearchSettings: "get: No settings file exists, new profile?" (new
 1622710492557	Marionette	INFO	Listening on port 53904
```

Comment:
We shouldn't need this file or the geckodriver binary (especially now you've transferred to a Chrome driver!) so we might want to remove it (and potentially add it to your .gitignore)